pkg/cmd/taskrun/list.go

```
@@ -60,7 +59,7 @@ tkn pr list -n foo \n",
 			trs, err := list(p, task)
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "Failed to list taskruns from %s namespace \n", p.Namespace())
-				return err
+				return nil
```

This should also return an error?